class Two_Neighbor_Values_Exception(Exception):

    def __init__(self):
        super().__init__()